src/nplinker/loader.py

```diff
@@ -439,22 +437,21 @@ def _load_genomics(self):
         raw_bgcs = antismash_bgcs + self.mibig_bgcs
 
         # Step 2: load all GCF objects
+        # TODO: create a config for "bigscape_cluster_file" and discard "bigscape_dir" and "bigscape_cutoff"?
         bigscape_cluster_file = (
             Path(self.bigscape_dir) / "mix" / f"mix_clustering_c0.{self._bigscape_cutoff:02d}.tsv"
         )
-        bigscape_gcf_list = BigscapeGCFLoader(bigscape_cluster_file).get_gcfs()
-        raw_gcfs = bigscape_gcf_list
+        raw_gcfs = BigscapeGCFLoader(bigscape_cluster_file).get_gcfs()
 
         # Step 3: assign Strain object to BGC.strain
         map_strain_to_bgc(self.strains, raw_bgcs)
 
         # Step 4: assign BGC objects to GCF.bgcs
         map_bgc_to_gcf(raw_bgcs, raw_gcfs)
 
-        # Step 5: get clean GCF objects, BGC objects and Strain objects
+        # Step 5: get GCF objects and their BGC members
         self.gcfs = raw_gcfs
         self.bgcs = get_bgcs_from_gcfs(self.gcfs)
-        self.strains = get_strains_from_bgcs(self.bgcs)
```

Comment:
Now `self.strains` is assigned in `_load_strain_mappings()`, correct?

Reply:
It's more complex than that. After loading from `strain mapping`, it's also needed to filter out user's specified strains and then get common strains from both sides (genomics and metabolomics). I'm just working on that, and you will see that in coming PRs.